content/documentation/glossary/index.adoc

```diff
@@ -0,0 +1,207 @@
+
+== Rate Interval
+
+It's an amount of time. By Default in Kiali last 10 minutes.
+
+== ReplicaSet
+
+Ensures that a specified number of pod replicas are running at any one time
+
+== Rule
+
+It's an object that manages external access to the services in a cluster, typically HTTP
+
+== Service
+
+Pods are dynamically, they are mortal. A <<Service>> is an abstraction which defines a logical set of Pods and a policy by which to access them.
+A Service is determined by a <<Label>>
+
+=== Service Endpoint
+
+The network-reachable manifestation of a service. Service endpoints are exposed by workload instances. Not all services have service endpoints.
+
+=== Service Entry
+
+ServiceEntry enables adding additional entries into Istio’s internal service registry, so that auto-discovered services in the mesh can access/route to these manually specified services. A service entry describes the properties of a service (DNS name, VIPs ,ports, protocols, endpoints). These services could be external to the mesh (e.g., web APIs) or mesh-internal services that are not part of the platform’s service registry (e.g., a set of VMs talking to services in Kubernetes).
+link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#ServiceEntry[Istio Service Entry]
+
+=== Service Name
+
+It's the name of the <<Service>>
+
+
+== Source
+
+The downstream client of the <<Envoy>> proxy. Within the service mesh a source is typically a <<Workload>>, but the source for ingress traffic may include other clients such as a browser or mobile app.
+
+link:https://istio.io/help/glossary/#source[Istio Glossary]
+
+=== Source Services
+
+It's the origin service where something is generated.
+
+== Subset
+
+
+== VirtualService
+
+A VirtualService defines a set of traffic routing rules to apply when a host is addressed. Each routing rule defines matching criteria for traffic of a specific protocol. If the traffic is matched, then it is sent to a named destination service (or subset/version of it) defined in the registry.
+link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#VirtualService[Istio VirtualService]
+
+== Workload
+
+A binary deployed by operators to deliver some function in Istio. Workloads have names, namespaces, and unique ids. These properties are available in policy and telemetry configuration using the following attributes:
+
++ source.workload.name, source.workload.namespace, source.workload.uid
++ destination.workload.name, destination.workload.namespace, destination.workload.uid
+In Kubernetes, a workload typically corresponds to a Kubernetes deployment, while a workload instance corresponds to an individual pod managed by the deployment.
+
+A workload in Kiali is a <<Deployment>> or <<ReplicaSet>>
+
+link:https://istio.io/help/glossary/#workload[Istio Glossary]
+
+=== Workload Name
+
+It's the name of the <<Workload>>
+
+=== Workload Type
+
+It's the type of the <<Workload>>, now we only support <<Deployment>> type.
```

Comment:
I think this document needs some higher level sections to group explained concepts.
Right now there's an unordered list of all terms. For example it starts with Application which explains it's a label on a workload, but the explanations what is a workload and what is a label come much later in the same doc.